test/dlc_tests/ec2/tensorflow/inference/test_tensorflow_inference.py

```diff
@@ -180,6 +194,15 @@ def host_setup_for_tensorflow_inference(serving_folder_path, framework_version,
         local_scripts_path = os.path.join("container_tests", "bin", "tensorflow_serving")
         ec2_connection.run(f"mkdir -p {serving_folder_path}")
         ec2_connection.run(f"cp -r {local_scripts_path} {serving_folder_path}")
+        if is_neuron:
+            neuron_local_model = os.path.join("$HOME", "container_tests/bin/neuron_tests/simple")
```

Comment:
`os.path.join` can take any number of input args. So please change this to:
```suggestion
            neuron_local_model = os.path.join("$HOME", "container_tests", "bin", "neuron_tests", "simple")
```